index.js

```diff
@@ -67,7 +67,7 @@ function fastifyMiddie (fastify, options, next) {
       req.raw.ip = req.ip
       req.raw.ips = req.ips
       req.raw.log = req.log
-      req.raw.body = req.body
+      if (req.body) req.raw.body = req.body
```

Comment:
please move it as the latest added property, otherwise V8 cannot perform its magic.

Reply:
Thanks Matteo, very interesting, I wasn't aware.